app/assets/javascripts/action_plan.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("turbolinks:load", function() {
+  $("input[name='action-plan_prefix']").keyup(function(event) {
+    let prefix = event.target.value;
+
+    $("h3.action-plan_heading").each(function(index) {
+      $(this).children("span").remove();
```

Comment:
what do you think about always having the `span` element there in the html view and set the innerText value instead of removing/prepending it? it should be more performant

you can do something like

```
$('h3.action-plan_heading > span').each(function(span) {
  span.innerText = `${prefix}${prefix.trim() == "" ? "" : index + 1}`;
})
```

Reply:
@arielj That is okay by me, especially if it is more performant than my version. I will make that change.